vmdb/app/models/provider.rb

```diff
@@ -27,6 +42,12 @@ def my_zone
   end
   alias_method :zone_name, :my_zone
 
+  def refresh_ems
+    raise "no #{ui_lookup(:table => "provider")} credentials defined" if self.missing_credentials?
+    raise "#{ui_lookup(:table => "provider")} failed last authentication check" unless self.authentication_status_ok?
+    managers.each { |manager| EmsRefresh.queue_refresh(manager) }
```

Comment:
can we leverage `ExtManagementSystem#refresh_ems` ?

```ruby
def refresh_ems(reload = false)
  ExtManagementSystem.refresh_ems(manager_ids, reload)
end
```

not sure if you want reload (force the reload)